src/Symfony/Component/HttpFoundation/Request.php

```diff
@@ -630,7 +630,7 @@ public static function normalizeQueryString($qs)
         }
 
         parse_str($qs, $qs);
-        ksort($qs);
+        ksort($qs, SORT_STRING);
```

Comment:
Is it worth having a test for this?

Reply:
I would suggest to revert that one: it may have unwanted side effects.

Reply:
Ok, will revert.